backend/api/app/application/services/user_service.py

```diff
@@ -76,3 +107,152 @@ def verifySSID(self, ssid: str, ip_address: str):
             return True
         else:
             return False
+        
+    def logout_user(self, ssid: str, ip_address: str):
+            try:
+                if not ssid:
+                    return {"message": "SSID is required", "status": 400}
+
+                try:
+                    ses_id = uuid.UUID(ssid)
+                except ValueError:
+                    return {"message": "Invalid SSID format", "status": 400}
```

Comment:
SSID is not something user should have concerns about, it is Authorization logic and we should not expose that to user in any way, this should be 403 error for example, and this needs to be handled by Middleware, which will make this check very likely to be redundant.